### PR TITLE
Remove aggregation

### LINKED ISSUE #142: Disable server side aggregation on get
Now that aggregation is triggered by database hooks, the server side aggregation post get is bringing some confusion especially in run_elements where it is enable by default.
Unless it is needed elsewhere,  it should be disable completely or at least not unable by default.

### PR DESCRIPTION
**Pull request by tcezard**
This PR remove all server/database side aggregation applied on GET method.
It includes the server side aggregation on embedded elements called with
```python
get_documents('samples', embedded={'run_elements: 1}, aggregate=True) 
```
Or the database side aggregation called with:
```python
get_documents('aggregate/samples') 
```
I want to ensure that no code uses these calls before the PR is completed so let me know if you know of any.

The embedding without aggregation will still work
```python
get_documents('samples', embedded={'run_elements: 1}) 
```
still adds the run elements collection in the samples.

closes #142

### CODE REVIEW

File: rest_api/__init__.py [hunk 1]
```diff
@@ -1,23 +1,19 @@
 import eve
+import pymongo
```
[the remark this answers is not in the document]
Comment (tcezard) [reply rest_api/__init__.py]: removed

[the remark this answers is not in the document]
Comment (mwhamgenomics) [reply rest_api/__init__.py]: Still there

[the remark this answers is not in the document]
Comment (tcezard) [reply rest_api/__init__.py]: Forgot to push 😊

**Comment by lukbut** [comment]
Validation ID: VB7

**Pull request merged**